conda_forge_tick/make_migrators.py

```diff
@@ -777,22 +796,13 @@ def initialize_migrators(
             python_nodes=python_nodes,
             graph=gx,
             pr_limit=PR_LIMIT * 4,
-            piggy_back_migrations=[
-                CondaForgeYAMLCleanup(),
-                Jinja2VarsCleanup(),
-                DuplicateLinesCleanup(),
-                PipMigrator(),
-                LicenseMigrator(),
-                CondaForgeYAMLCleanup(),
-                ExtraJinja2KeysCleanup(),
-                Build2HostMigrator(),
-                NoCondaInspectMigrator(),
-                PipWheelMigrator(),
-                MPIPinRunAsBuildCleanup(),
-                DependencyUpdateMigrator(python_nodes),
-                PyPIOrgMigrator(),
-                StdlibMigrator(),
-            ],
+            piggy_back_migrations=_make_mini_migrators_with_defaults(
+                extra_mini_migrators=[
+                    PipWheelMigrator(),
+                    DependencyUpdateMigrator(python_nodes),
+                    StdlibMigrator(),
+                ],
```

Comment:
I see it gets readded here anyway, but that just sounds to me like it should be in the default set? 🤔

Reply:
I tried to separate formatting versus recipe behavior changes, with the goal of future additions to the migrators preferring one over the other depending on context.

Reply:
OK, fair enough. 🤷